src/verify.c

```diff
@@ -930,7 +1001,16 @@ enum swupd_code verify_main(void)
 		}
 	}
 
-	ret = add_included_manifests(official_manifest, &subs);
+	ret = add_included_manifests(official_manifest, &all_subs);
+	if (cmdline_option_bundles) {
+		/* when using --bundles, the subs lists will be browsed
+		 * and validated twice, once for all subs and once for the
+		 * specific bundles subs, to avoid printing double warning
+		 * we need to mute the log during those operations */
+		log_mute(true);
+		ret = add_included_manifests(official_manifest, &bundles_subs);
```

Comment:
what real logging are you trying to avoid here?

Reply:
add_included_manifests calls add_subscriptions which is going to print this warning if a bundle is not found in the mom:
`warn("Bundle \"%s\" is invalid, skipping it...\n", bundle);`

So if we use a command like `swupd diagnose --bundles valid_bundle,invalid_bundle`

We are going to get this message twice:
```
Bundle "invalid_bundle" is invalid, skipping it...
Bundle "invalid_bundle" is invalid, skipping it...
```

So in some cases I think is useful to call a function without printing any of its output, specially if you are already printing something similar at a higher level, so there is where muting the log comes handy. Is like having a parameter for the function to enable it to run silently.